worlds/bomb_rush_cyberfunk/__init__.py

```diff
@@ -0,0 +1,208 @@
+
+        if self.options.starting_movestyle == StartStyle.option_skateboard:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.progression
+
+        if self.options.starting_movestyle == StartStyle.option_inline_skates:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.progression
+        
+        if self.options.starting_movestyle == StartStyle.option_bmx:
+            self.item_classification[BRCType.BMX] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.BMX] = ItemClassification.progression
+
+
+    def create_items(self):
+        self.options.total_rep.round_to_nearest_step()
+        rep_counts = self.options.total_rep.get_rep_item_counts(self.multiworld.random, 86)
+        #print(sum([8*rep_counts[0], 16*rep_counts[1], 24*rep_counts[2], 32*rep_counts[3], 48*rep_counts[4]]), \
+        #    rep_counts)
+
+        pool = []
+
+        for item in item_table:
+            if "REP" in item["name"]:
+                count: int = 0
+
+                if item["name"] == "8 REP":
+                    count = rep_counts[0]
+                elif item["name"] == "16 REP":
+                    count = rep_counts[1]
+                elif item["name"] == "24 REP":
+                    count = rep_counts[2]
+                elif item["name"] == "32 REP":
+                    count = rep_counts[3]
+                elif item["name"] == "48 REP":
+                    count = rep_counts[4]
+
+                if count > 0:
+                    for _ in range(count):
+                        pool.append(self.create_item(item["name"]))
+            else:
+                pool.append(self.create_item(item["name"]))
+
+        self.multiworld.itempool += pool
+
+
+    def get_stage(self, stage: BRCStage, player: int) -> Region:
+        return self.multiworld.get_region(region_names[stage], player)
+
+
+    def create_regions(self):
+        world = self.multiworld
+        player = self.player
+
+        menu = Region("Menu", player, world)
+        world.regions.append(menu)
+
+        for _, n in region_names.items():
+            world.regions += [Region(n, player, world)]
+
+        menu.add_exits({"Hideout": "New Game"})
+
+        for r, e in region_exits.items():
+            exits = {region_names[ex] for ex in region_exits[r]}
+            self.get_stage(r, player).add_exits(exits)
+
+        for index, loc in enumerate(location_table):
+            stage: Region = self.get_stage(loc["stage"], player)
+            stage.add_locations({loc["name"]: base_id + index})
+            if self.options.skip_polo_photos and "Polo" in loc["name"]:
+                self.options.exclude_locations.value.add(loc["name"])
```

Comment:
Is there a reason these are excluded instead of being entirely removed?

Reply:
You could also expand the option so the player can decide whether they are excluded or removed entirely.

Reply:
I think I'd rather not do that since I feel like having both "Excluded" and "Removed" as options would probably be confusing to people who aren't familiar with Archipelago. With that said though, I think you've changed my mind. I'll make the option remove the locations, and then people who know advanced YAML options can just add them to `exclude_locations` themselves instead if that's what they want.